.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE_TEAM.md

```diff
@@ -0,0 +1,29 @@
+<!-- 
+    This is a custom pull request template for FerretDB Inc. engineers.
+    It is more complex than the default template, but it contains some additional
+    information that is useful for team members.
+-->
+
+## Description
+
+This PR closes #{issue_number}.
+
+<!-- 
+    Write a short description to explain changes that are not mentioned in the initial issue.
+    What were the reasons for those changes?
+    Which decisions did you make and why? 
+    What else should reviewers know about your changes? 
+-->
+
+## Readiness checklist
+
+<!-- 
+    If you want your changes to be merged quickly,
+    please follow CONTRIBUTING.md.
+-->
+
+* [ ] I set assignee, reviewers, labels, milestone, project and sprint.
+* [ ] I added tests for new functionality or bugfixes.
```

Comment:
May we rely on code coverage here? Because it's a reason for me to request changes. And it may induce more code to write often.